app/database/migration/server/index.ts

```diff
@@ -22,6 +22,7 @@ export default schemaMigrations({migrations: [
                     {name: 'files', type: 'string'},
                     {name: 'root_id', type: 'string', isIndexed: true},
                     {name: 'metadata', type: 'string', isOptional: true},
+                    {name: 'create_at', type: 'number'},
```

Comment:
Sanity check: Here we are reusing a migration. Is that intended? I understand it is, because this will be the migration for the whole feature, and not something already in main, right?

Reply:
Yup, that's right, the whole feature needs migration (adding the scheduled post table to the local db), this is just a small part which was left.